src/Project.elm

```diff
@@ -10,13 +9,18 @@ type Owner
 
 
 type alias Project a =
-    { a | owner : Owner, name : FQN, hash : Hash }
+    { a | owner : Owner, name : FQN }
```

Comment:
Remove the Hash field for now to make it easier to mock (creating a `Hash` returns a `Maybe`)—it'll be back in a later iteration.